Rechtschreibung, Formulierungen, hinzugefügt: triviale Formen des Spo…

woerde opened the pull request:
…nsorings

Habs mal so überarbeitet, wie ich es gut fände. Was meint Ihr?

woerde committed:
Rechtschreibung, Formulierungen, hinzugefügt: triviale Formen des Sponsorings

Habs mal so überarbeitet, wie ich es gut fände. Was meint Ihr?

kgbvax commented:
Ich finde es ziemlich gut. Wenn wir "konkrete Formen" nennen wollen (was ich auch gut finde) müsste die Liste noch etwas erweitert werden (um nicht technische aka Propaganda Optionen). Das ist aber leicht.

woerde committed:
Bandwurmsatz gebändigt

Dabei aber auch die Aussage verändert. Das Wort "emotional"darf gerne zerrissen werden :-)

ghost commented:
Wie steht's um diesen Änderungsvorschlag?

MPW1412 commented:
Es macht es insgesamt etwas runder; womit ich mich nicht anfreunden kann ist die neue Zeile 18:

„Viele aktive Freifunker investieren große Mengen Zeit und auch Geld in Freifunk und fühlen sich daher auch emotional sehr verbunden. “

Besser: 

„Viele aktive Freifunker investieren viel Zeit und auch Geld in Freifunk und fühlen sich daher sehr mit dem Projekt verbunden.“

Emotional verbunden geht irgendwie gar nicht :).

Vielen Dank für die ganze Mühe zur Überarbeitung!

woerde commented:
Viel besser!

Vorschlag: ich arbeite die Änderungen gleich  ein und erweitere die Liste, wie von kgbvax gewünscht.

Es geht hier ja um Detailverbesserungen. Wie ist dann der korrekte Weg, das beschlossene Dokument zu ändern, ohne jemanden zu übergehen und ohne da jetzt zu viel Arbeit zu investieren?

woerde committed:
Einarbeitung der Anmerkungen

mehr konkrete Beispiele für Fördermöglichkeiten sind mir nicht eingefallen. Gerne ergänzen ...

kgbvax commented:
Solange es eher redaktionelle Änderungen sind können wir das glaube ich einfache machen und das Ergebnis hochhalten. Ich muss das nochmal auf Papier lesen dann merge ich das als ersten Schritt.

woerde commented:
Ich werde die aktuelle Arbeitsversion heute mal verwenden, um einem potentiellen Großsponsor in Ostbevern unsere Regeln nochmal zum Nachlesen zu geben. Wäre toll, wenn Du, @kgbvax, trotzdem bei Gelegenheit nochmal drüber lesen und redigieren könntest!

ghost commented:
Wie schaut es hiermit aus?

MPW1412 commented:
Letzter Stand steht ja hier, @kgbvax liest sich das durch und mergt es. Ist wohl in den Hintergrund geraten. Ich finde das Dokument nicht mehr so relevant.

ghost commented:
War auch eher die unterschwellige Frage, ob wir das hier schließen können...

MPW1412 commented:
Was meint denn @woerde dazu?

woerde commented:
Ich habe das Dokument mit den von mir vorgeschlagenen Änderungen bisher immer ganz gerne verwendet. Mir ist nicht ganz klar, warum anderswo kein Bedarf besteht, aber offensichtlich ist das so.

Mein Vorschlag wäre, die Mini-Verbesserungen von meiner Seite entweder einfach zu übernehmen, falls jemand bestätigen kann, dass der Rest noch so passt. Oder halt das Dokument zu den Akten zu legen, falls kein Bedarf besteht.

Ich habe auch nochmal überlegt, warum ich dieses Zettelchen mag. Ich kann nicht so gut Zeit finden, all die Diskussionen zur Meinungsbildung im Detail zu verfolgen. Daher hilft mir ein Zettel dieser Art (auch zu anderen zentralen Themen), wenn dort "unsere" (=alle Freifunker im Münsterland inklusive mir) Positionen und Handlungsempfehlungen zusammengefasst sind. Quasi, um mich selber auf dem aktuellen Stand zu halten. Das kann ein Wiki vielleicht sogar besser leisten, denke ich. Der andere Anwendungsfall ist, dass ich dieses Dokument gerne zusammen mit dem Protokoll eines Projekt-Kickoffs verschicke, um nochmal schriftlich die Regeln zu kommunizieren, nach denen Freifunk gespielt wird. Sonst wird der Punkt "Förderung der Infrastruktur" gerne schnell von potentiellen Förderern wieder vergessen. Und später heißt es "och, das hatten wir ja überhaupt nicht eingeplant"

kgbvax commented:
Wieso nicht mehr relevant? Nur weil gerade nichts dergleichen anliegt?

ghost commented:
Ich nerve mal weiter. Soll der Pull Request angenommen oder abgelehnt werden? Was spricht für eine Nicht-Entscheidung dessen?
